src/ops/constant.rs

```diff
@@ -308,8 +307,9 @@ impl_box_clone!(CustomConst, CustomConstBoxClone);
 
 // Don't derive Eq here - the yaml could contain floats etc.
 // (Perhaps we could derive Eq if-and-only-if "typ.tag() == TypeTag::Hashable"!)
-#[derive(Debug, Clone, PartialEq, serde::Serialize, serde::Deserialize)]
-struct CustomSerialized {
+#[derive(Debug, Clone, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
```

Comment:
Whoa - do we need *this* `Eq`? See comment just above

Reply:
CustomSerialized is now in TypeArg -> TypeArg is in TypeArgError -> TypeArgError can't be Eq  , which is probably why I originally removed it